cluster/examples/kubernetes/rook-cluster.yaml

```diff
@@ -10,7 +10,9 @@ metadata:
   namespace: rook
 spec:
   versionTag: master
-  dataDirHostPath:
+  # the path on the host where configuration files will be persisted.
+  # Important: if you reinstall the cluster, make sure you delete this directory from each host or else the mons will fail to start on the new cluster.
```

Comment:
Can you add that is left empty, it will use emptyDir?

Reply:
will do